### PR TITLE
Clarifications, Annotations, Cleanup

### PR DESCRIPTION
**Pull request by julianboehm**
Some changes that accumulated while reviewing the code in preparation for `continuous-billing` branch.  
Might be a good idea to merge this before it creates unneeded merging conflicts.

### CODE REVIEW

File: utility-js/Utility.js [hunk 1]
```diff
@@ -61,22 +58,6 @@ class Utility {
     return this._householdExists(hhAddress) ? this.households[hhAddress] : {};
   }
 
-  /**
```
Comment (petscheit) [inline utility-js/Utility.js]: the api endpoint` /network `in `ned-server/index.js` has to be changed as well, as it calls these getters

**Comment by petscheit** [comment]
Other then the small ned-server change that is still needed, looks good to me

File: utility-js/Utility.js (hunk 1 above)
Comment (Gjum) [reply utility-js/Utility.js]: good catch! fixed in a91bea6.

Comment (julianboehm) [reply utility-js/Utility.js]: Good catch! Fixed in a91bea6.

**Pull request merged**